.gitignore

```diff
@@ -23,5 +23,7 @@ build
 *.zip
 *.egg-info
 *.xml
+results*
```

Comment:
this are a bit too general names, maybe put them inside visualization folder by default and ignore there?